repositories/createAnswerReaction.ts

```diff
@@ -0,0 +1,40 @@
+import * as firebase from "firebase";
+import Answer from "../models/Answer";
+import ReactionType from "../models/ReactionType";
+import User from "../models/User";
+
+export type CreateAnswerReaction = (params: {
+  answer: Answer;
+  reactionType: ReactionType;
+  user: User;
+}) => Promise<void>;
+
+export function createCreateAnswerReaction({
+  firebaseApp
+}: {
+  firebaseApp: firebase.app.App;
+}): CreateAnswerReaction {
+  return async ({ answer, reactionType, user }) => {
+    await firebaseApp
+      .firestore()
+      .collection("post")
```

Comment:
posts?

Reply:
That's right. Thanks! I really wanna write tests for that...